spec/acceptance/nodesets/default.yml

```diff
@@ -1,18 +1,24 @@
----
 HOSTS:
-  windows2012-64-1:
-    pe_dir:
-    pe_ver:
-    pe_upgrade_dir:
-    pe_upgrade_ver:
-    hypervisor: vmpooler
-    platform: windows-2012-64
-    ruby_arch: x64
-    template: win-2012-x86_64
+  centos7-master:
```

Comment:
Do we want a master in the default nodeset here?  Could be a commit after this work...

Reply:
This file is coming from modsync so it probably shouldn't be modified.

Reply:
Also - is this why `sql_version: 2012` appears in here?